versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/AbstractDependencyUpdatesReport.java

```diff
@@ -80,6 +81,17 @@ public abstract class AbstractDependencyUpdatesReport extends AbstractVersionsRe
     @Parameter(property = "processDependencyManagementTransitive", defaultValue = "true")
     protected boolean processDependencyManagementTransitive;
 
+    /**
+     * <p>Include dependencies with version set in a parent or in a BOM.</p>
+     * <p>This is similar to {@code processDependencyManagementTransitive}, but will
+     * report updates on dependencies.</p>
+     * <p>Default is {@code false}.</p>
```

Comment:
When we use `defaultValue` in `@Parameter` is will be added to docs.
By the way has different value .... doc and parameter

Reply:
I get your point, but I did not change `processDependencyManagementTransitive` in this PR.

On the other hand, this (initializing the properties) immensely helps with unit testing.